docs/contrib/DatabaseMath.md

```diff
@@ -23,50 +32,58 @@ negative number looks like a very very big number. So this is a very bad
 idea and leads to bugs.
 
 Note that calculating an area with `int` values is problematic. An
-`int * int` does not fit in an `int`. My suggestion is to use `int64_t`
-in this situation. Although `long` "works", its size is implementation-dependent.
+`int * int` does not fit in an `int`. **Our suggestion is to use `int64_t`
+in this situation.** Although `long` "works", its size is implementation-dependent.
+
+### Why not double?
 
-Unfortunately, I have seen multiple instances of programs using a
-`double` for distance calculations. A double is 8 bytes, with 52 bits
-used for the mantissa. So, the largest possible integer value that can be
-represented without loss is 5e+15, 12 bits less than using an `int64_t`.
-Doing an area calculation on a large chip that is more than
-`sqrt(5e+15) = 7e+7 DBU` on a side will overflow the mantissa and truncate the
-result.
+It has been noticed that some programs use `double` to calculate distances. 
```

Comment:
Some of these are programmatic guidelines like in this case of using double to calculate distances. I suggest making these as a Guidelines section and adding a bulleted list that looks like this:
. Don't use double to calculate distances, Use int64_t instead. in one or 2 sentences explain the impact...